### DISCUSSION TITLE
Is there any open source project built based on abp.io

### DISCUSSION THREAD

**Discussion by rajasingh012** [abpframework/abp#7432] [General]
Is there any open source project built based on abp.io, if yes can you provide the link, and what the challenges you faced, and how it was resolved. 

Is the template is good for small team(3-4 developers)

**Comment by ananck** [comment]
[ABP-MicroService](https://github.com/WilliamXu96/ABP-MicroService)

**Comment by rajasingh012** [comment]
[SoMall](https://github.com/jerrytang67/SoMall)

**Comment by ebicoglu** [comment]
Anyone has a working live website built on top of ABP?

**Reply by go2ismail** [reply]
Hi, live ABP based app that I create as a demo example: https://inventory-pro.indotalent.com/   https://abp-adminlte-demo.indotalent.com/Dashboard  https://github.com/go2ismail/ABP-Framework-Theme-Using-AdminLTE3-Template
